cves/CVE-2019-19045.yml

```diff
@@ -0,0 +1,503 @@
+CVE: CVE-2019-19045
```

Comment:
We don't need to copy these files over - make sure you're just editing the kernel/ files.

Reply:
Yes, I had it just in case and forgot to delete them. Thank you for pointing it out!